web-common/src/components/data-graphic/marks/ChunkedLine.svelte

```diff
@@ -84,6 +95,37 @@ Over time, we'll make this the default Line implementation, but it's not quite t
    */
   let lineThickness = createAdaptiveLineThicknessStore(yAccessor);
   $: lineThickness.setData(data);
+
+  /** Get center aligned clips for segments */
+  function getClipPathParams(segment) {
```

Comment:
The new function was being used to extend the line beyond the center so that it covers the full time bucket but I suppose it's more confusing than helpful.

![image](https://github.com/rilldata/rill/assets/4402679/15d82f50-30c6-4a49-bca8-e1a224bdc7ee)

Will revert back to the old setup.